common/pom.xml

```diff
@@ -41,7 +41,7 @@
     <dependency>
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka_${scala.binary.version}</artifactId>
-      <version>0.8.2.1</version>
+      <version>0.10.2.2</version>
```

Comment:
This is the version works for scala 2.12? Have you tested with the streaming benchmark. I am wondering if it still works.

Reply:
Sadly we don't have streaming benchmark auto testing : (

Reply:
I confirm that streaming benchmark are not working in Java 11.

In all `dataGen.sh` files in streaming workloads could not even start Kafka server since [GC Combinations flags](https://github.com/Intel-bigdata/HiBench/blob/master/bin/workloads/streaming/fixwindow/prepare/dataGen.sh#L29) like `UseParNewGC` are not supported after Java 8 as mentioned in [JEP 214](https://openjdk.java.net/jeps/214):
```
Here is a detailed summary of the flags and flag combinations that will stop working:

DefNew + CMS       : -XX:-UseParNewGC -XX:+UseConcMarkSweepGC
ParNew + SerialOld : -XX:+UseParNewGC
ParNew + iCMS      : -Xincgc
ParNew + iCMS      : -XX:+CMSIncrementalMode -XX:+UseConcMarkSweepGC
DefNew + iCMS      : -XX:+CMSIncrementalMode -XX:+UseConcMarkSweepGC -XX:-UseParNewGC
CMS foreground     : -XX:+UseCMSCompactAtFullCollection
CMS foreground     : -XX:+CMSFullGCsBeforeCompaction
CMS foreground     : -XX:+UseCMSCollectionPassing
```

I believe that this should be part o f a different PR/effort, in order to have both compatibility (Java 8 and greater versions) like detecting `$JAVA_BIN` in the [command used to launch server](https://github.com/Intel-bigdata/HiBench/blob/master/bin/workloads/streaming/fixwindow/prepare/dataGen.sh#L33) to know which flags to use.

Reply:
@luisfponce , I think this is going to break the streaming benchmark even in JAVA 8. We can not simple upgrade the kafka dependency from 0.8.2.1  to 0.10.2.2. Because in the sparkbench - streaming module, we are still using spark-streaming-kafka-0-8_2.11. 

For scala-2.11 and scala-2.10, can we still use kafka 0.8.2.1?  We can specify them in the scala-2.11 and scala-2.10 profiles.

Reply:
Shure @carsonwang.
I'll perform the pertinent changes to keep using in 0.8.2.1 to do not break streaming in scala 2.11 and 2.10.

Reply:
@carsonwang I created #589

Reply:
@carsonwang done.
Reverted streaming-kafka to 0.8.2.1